nixos/modules/misc/ids.nix

```diff
@@ -483,6 +484,7 @@
       cfdyndns = 227;
       pdnsd = 229;
       octoprint = 230;
+      radicale = 231;
```

Comment:
Okay, the policy seems to be that users/groups with same name should have the same uid/gid also, if possible. The "gap" here is really a block of reserved/unused gids.